students/ikempcode33/Lesson 2/charges_calc.py

```diff
@@ -0,0 +1,115 @@
+# $ python -m pdb charges_calc.py -i source.json -o out.json -d 0
+'''
+Returns total price paid for individual rentals
+'''
+import logging
+import argparse
+import json
+import datetime
+import math
+import sys
+
+
+def setup_logger(level):
+    """sets up logging properties"""
+    logging_levels = {0: logging.CRITICAL,
+                      1: logging.ERROR,
+                      2: logging.WARNING,
+                      3: logging.INFO,
+                      4: logging.DEBUG}
+
+    try:
+        debug_level = logging_levels.get(int(level))
+    except KeyError:
+        logging.critical("Error: Please set debug level to 0,1,2,3 or 4")
+        sys.exit()
+    # set up format, handler, consol for log messages, and root handler for logging
+    log_format = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+    formatter = logging.Formatter(log_format)
+    log_file = datetime.datetime.now().strftime("%Y-%m-%d") + ".log"
+    file_handler = logging.FileHandler(log_file)
+    file_handler.setFormatter(formatter)
+    file_handler.setLevel(debug_level)
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(logging.DEBUG)
+    console_handler.setFormatter(formatter)
+    logger = logging.getLogger()
+    logger.setLevel(debug_level)
+    logger.addHandler(file_handler)
+    logger.addHandler(console_handler)
+
+    loggingSetup = "Logging set up complete."
```

Comment:
Just to respect PEP8 convention, this one should be called _logging_setup_ (snake case versus camel case).